src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -172,6 +180,13 @@ export class UserFormComponent implements OnInit {
     }
   }
 
+  checkPassword() {
+    const password = this.userForm.get('password').value;
+    const passwordStrength = this.userChangePasswordService.checkPasswordComplexity(password);
```

Comment:
Simply for better code reading.
```suggestion
    const passwordStrength: [
      string,
      string
    ] = this.userChangePasswordService.checkPasswordComplexity(password);
```
or better use
```suggestion
    [this.passwordStrengthLevel, this.passwordStrengthDescription] = this.userChangePasswordService.checkPasswordComplexity(password);
```
Don't forget to remove the following lines then.
```
    this.passwordStrengthLevel = passwordStrength[0];
    this.passwordStrengthDescription = passwordStrength[1];
```